submitted_models/csiro_data61_ozbot_atr_sensor_config_1/launch/spawner.rb

```diff
@@ -0,0 +1,112 @@
+def spawner(_name, _modelURI, _worldName, _x, _y, _z, _roll, _pitch, _yaw)
+  <<-HEREDOC
+  <plugin name="ignition::launch::GazeboFactory"
```

Comment:
Please use `<spawn>` instead of `<plugin ...`.

Reply:
See https://github.com/TJMolnar/subt_csiro_data61_ozbot_atr/pull/1